community/graphdb-api/src/main/java/org/neo4j/graphdb/GraphDatabaseService.java

```diff
@@ -55,6 +55,16 @@
      */
     Node createNode();
 
+    /**
+     * Creates a new node and returns it id.
+     * Please note: Neo4j reuses its internal ids when
+     * nodes and relationships are deleted, which means it's bad practice to
+     * refer to them this way. Instead, use application generated ids.
+     *
+     * @return the created nodes id.
+     */
+    Long createNodeId();
```

Comment:
:scream:

Reply:
its strange to see more than one person to find out about that method almost simultaneously